frontend/dockerfile/dockerfile_test.go

```diff
@@ -6764,6 +6765,102 @@ ARG BUILDKIT_SBOM_SCAN_STAGE=true
 	require.Equal(t, 1, len(att.LayersRaw))
 }
 
+func testLintWarnings(t *testing.T, sb integration.Sandbox) {
+	integration.SkipOnPlatform(t, "windows")
+	f := getFrontend(t, sb)
+
+	// empty line in here is intentional to cause line continuation warning
+	dockerfile := []byte(`
+# set of commands which should cause warnings
+FROM scratch as BadStageName
+Copy Dockerfile .
+COPY Dockerfile \
+
+.
+
+# set of commands which should not cause warnings
+FROM scratch as base
+copy Dockerfile .
```

Comment:
I think this is still warning:

No warning:
```
FROM scratch AS base
COPY
```

```
from scratch as base
copy
```

With warning:

```
FROM scratch as base
COPY
```

```
FROM scratch AS base
copy
```

Reply:
Ended up being easiest to make this its own lint rule!